packages/suite/src/components/suite/section/sectionStyles.tsx

```diff
@@ -31,10 +34,24 @@ export const ActionSelect = styled((props: SelectProps) => <Select {...props} si
     }
 `;
 
-export const ActionButton = styled((props: ButtonProps) => <Button {...props} size="small" />)`
-    min-width: 140px;
-    margin: 4px 0 4px 4px;
-
+export const ActionButton = styled(
+    ({
+        tooltipContent,
+        children,
+        ...buttonProps
+    }: { tooltipContent?: ReactNode } & ButtonProps) => (
+        <Tooltip content={tooltipContent}>
```

Comment:
`isActive` by `isActive={Boolean(tooltipContent)}` or not?

Reply:
Rather not, I used new `isTooltipActive` prop